07_cmd_lab/cmdlab.txt

```diff
@@ -3,11 +3,16 @@ Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
   - your current shell?
+echo $0
   - your default shell?
+echo $SHELL
   - a list of all shells currently installed on the machine?
-
+cat /etc/shells
 2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
-
+The file describes user login accounts for the system.
+Each line of the file describes a single user and contains 7 fields:
+name:password:UID:GID:GECOS:directory:shell
 3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
-
+ls -S /etc
 4. Print current date/time formatted as: "Sun 21/03/2010, 14:30". Hint: read "man date" and search for the correct format string.
+date +%a' '"%d/%m/%Y"','' '"%H:%M"
```

Comment:
that works :) but too many quotes.. you can just do:
```date "+%a %d/%m/%Y,  %H:%M"```
you can actually put any text in the format string, try:
```date "+%a %d/%m/%Y look! I can just put text here  %H:%M"```